docs/install/collecting-logs/README.md

```diff
@@ -0,0 +1,87 @@
+This document describes how to set up [Fluent Bit](https://docs.fluentbit.io/),
+a log processor and forwarder, to collect your kubernetes logs in a central
+directory. This is not required for running Knative, but can be helpful with
+[Knative Serving](../serving), which will automatically delete pods (and their
+associated logs) when they are no longer needed. Note that Fluent Bit support
+exporting to a number of other log providers; if you already have an existing
+log provider (for example, Splunk, Datadog, ElasticSearch, or Stackdriver), then
+you may only need the second part of setting up and configuring log forwarders.
+
+Setting up log collection consists of two pieces: running a log forwarding
+DaemonSet on each node, and running a collector somewhere in the cluster (in our
+example, we use a StatefulSet which stores logs on a Kubernetes
+PersistentVolumeClaim, but you could also use a HostPath).
+
+## Setting up the collector
+
+It's useful to set up the collector before the forwarders, because you'll need
+the address of the collector when configuring the forwarders, and the forwarders
+may queue logs until the collector is ready.
+
+![System diagram: forwarders and co-located collector and nginx](system.svg)
+
+<!-- yuml.me UML rendering of:
+[Forwarder1]logs->[Collector]
+[Forwarder2]logs->[Collector]
+
+// Add notes
+[Collector]->[shared volume]
+[nginx]-[shared volume]
+-->
+
+The [`fluent-bit-collector.yaml`](./fluent-bit-collector.yaml) defines a
+StatefulSet as well as a Kubernetes Service which allows accessing reading the
```

Comment:
```suggestion
StatefulSet as well as a Kubernetes Service which allows accessing and reading the
```

Not sure if you just wanted "accessing" or "reading", prob just "accessing"

Reply:
Done.